### PR TITLE
created placeholders for profile page

### PR DESCRIPTION
**Pull request by KaOus22**
Created a rough draft of the profile page. This will be dynamic so once we have the db completely scaffolded there will not be images hardcoded in the html like this.

### CODE REVIEW

**Comment by brett-buskirk** [comment]
This looks good to me, @KaOus22! I'll push it through.

File: rcjourney/rcapp/templates/dashboard.html [hunk 1]
```diff
@@ -23,8 +30,9 @@
       <li class="nav-item">
         <a class="nav-link" href="{% url 'listings' %}">Listings</a>
       </li>
+
       <li class="nav-item">
-        <a class="nav-link" href="{% url 'editprofile' %}">Profile</a>
```
Comment (brett-buskirk) [inline rcjourney/rcapp/templates/dashboard.html:27]: This was intentional. The dashboard is the only place a user can go and edit their profile from. This should be changed back to `editprofile`.

**Review by brett-buskirk — CHANGES_REQUESTED** [review]
Looks good other than where you changed the link in the dashboard. The link was supposed to be `editprofile` and not `profile`. The intention is that a user can edit their own profile from the dashboard. The `profile` link is for when they click on the post of another user and it takes them to that user's profile page.

File: rcjourney/rcapp/templates/dashboard.html (hunk 1 above)
Comment (KaOus22) [reply rcjourney/rcapp/templates/dashboard.html:27]: Upon pushing the changes up I got a message that says the repo was changed..

**Review by brett-buskirk — APPROVED** [review]

**Pull request merged**